javascript/src/main/pages/Home/Home.js

```diff
@@ -37,7 +37,15 @@ const Home = () => {
             <div className="text-left">
                 <h5>Welcome to the UCSB Courses Search App!</h5>
                 <BasicCourseSearchForm setCourseJSON={setCourseJSON} fetchJSON={fetchBasicCourseJSON} />
-                <Button><CSVLink style={{color: "white"}} headers={courseHeaders} data={courseJSON.classes} filename = {"CourseTable.csv"}>Download CSV</CSVLink></Button>
+                <Button>
+                    <CSVLink
+                        style={{color: "white", margin: "1rem 0"}}
```

Comment:
I'm not noticing any difference between the "Download CSV" button in your change (on the left) and the button in the current production app / main branch (on the right). See the screenshot below.

![image](https://user-images.githubusercontent.com/34492835/108981696-d6956980-7641-11eb-8403-ed509df5ac2d.png)

It looks like you're adding the margin to the wrong component - instead of adding it to the link, we should be adding it to the button. Can we make this fix? It should look something like this:

![image2](https://user-images.githubusercontent.com/34492835/108982206-65a28180-7642-11eb-9470-c6536c227e75.png)